packages/dev/docs/pages/releases/2024-07-22.mdx

```diff
@@ -0,0 +1,153 @@
+
+Our color components are also moving to RC! Focus management within Toasts have also been greatly improved, making announcements from screen readers clearer and more consistent. Thanks to [@thmsfchtnr](https://github.com/thmsfchtnr), press events now include coordinates, enabling interactions like our new [animated ripple button example](/react-aria/examples/ripple-button.html). And finally, in preparation for React 19, this release is now compatible with React 19 RC.
+
+Thank you to our ever growing list of contributors! ❤️
+
+### API updates
+Because of various breaking changes, several packages are being bumped a major version, specifically @react-aria/virtualizer , @react-stately/virtualizer , and @react-stately/layout. While these packages were undocumented and not yet intended for public use, we know this is not always the case, so please review the updated code carefully.
+
+## Enhancements
+
+- Breadcrumbs
+  - Support renderProps in Breadcrumbs - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/6440) - [PR](https://github.com/adobe/react-spectrum/pull/6489)
+- Date and time
+  - Refactor date segment width to take the space of placeholder or value  - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/6562)
+- Collections
+  - Support grid layout and tab navigation in GridList - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6486)
+  - Selection checkbox should always toggle regardless of `selectionBehavior` - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6526)
+  - Refactor collection internals in preparation for public API - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6608)
+  - Move RAC collections implementation into new package - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6640), [PR](https://github.com/adobe/react-spectrum/pull/6725)
+  - Expose `buttonLabelBehavior` prop in ActionBar - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/6708)
+- Icons
+  - The icons packages we depend on have been re-licensed to the more permissive Apache 2.0 license - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6735)
```

Comment:
my previous comment got lost. maybe we should be specific that this is for Spectrum.

```suggestion
  - Spectrum icons have been re-licensed as Apache 2.0 - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/6735)
```